packages/settings/store/middleware.js

```diff
@@ -9,6 +9,7 @@ const settings = store => next => (action) => {
   switch (action.type) {
     case actionTypes.networkConfigSet:
       // store.dispatch(pricesRetrieved());
```

Comment:
When will this be reinstated?

Reply:
This change was made by @isalga in the general refactor PR